server/routes/appointments/video-link-booking/middleware/initialiseJourney.ts

```diff
@@ -22,8 +23,39 @@ export default ({ bookAVideoLinkService, prisonService }: Services): RequestHand
 
     const prisoner = await prisonService.getInmateByPrisonerNumber(mainAppointment?.prisonerNumber, user)
 
+    const locationIds = await Promise.all(
+      _.uniq([mainAppointment?.prisonLocKey, preAppointment?.prisonLocKey, postAppointment?.prisonLocKey])
+        .filter(Boolean)
+        .map(code => prisonService.getInternalLocationByKey(code, user)),
+    ).then(locations => locations.map(l => l.locationId))
+
+    const existingVlbAppointments = await activitiesService
+      .searchAppointments(
+        user.activeCaseLoadId,
+        {
+          appointmentType: 'INDIVIDUAL',
+          startDate: mainAppointment.appointmentDate,
+          endDate: mainAppointment.appointmentDate,
```

Comment:
I think this end date is now redundant.

I think it looks like the same as the change made to the activities API yesterday i.e. is has been removed.